### PR TITLE
Moving Campaign and Project descriptions to Sir Trevor

### PR DESCRIPTION
**Pull request by annalee**
Switches the description fields on Projects and Campaigns to use Sir Trevor instead of TinyMCE.

Sir Trevor stores markdown inside json, which isn't playing nicely with manage.py's dumpdata command. I've manually edited the test fixture while I figure out a better fix for the problem. Running the included data migration will get  the current test data working.

Still need:
[ ] CSS fix to keep the new fields from doing ugly things to the layout
[ ] image resizing
[ ] 508-compliant image blocks for Sir Trevor
[ ] an image library that works with Sir Trevor, if we can get it

### CODE REVIEW

File: peacecorps/peacecorps/templatetags/markdown.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from markdown import markdown
+from django_jinja import library
+
+@library.filter
+def markdown(text):
+    return markdown(text)
```
Comment (cmc333333) [inline peacecorps/peacecorps/templatetags/markdown.py]: Is this file still needed? I think the `.html` stuff doesn't use jinja

File: peacecorps/peacecorps/urls.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
 urlpatterns = patterns(
     '',
+    url(r'^sirtrevor/', include('sirtrevor.urls')),
```
Comment (cmc333333) [inline peacecorps/peacecorps/urls.py]: Is this only needed for the admin? If so, can we conditionally add it?

**Comment by cmc333333** [comment]
Nice work on the migrations. I think you may have missed an update to `requirements.txt`. Does anything need to be added to `INSTALLED_APPS`? Can we remove all references to tinymce?

File: peacecorps/peacecorps/templatetags/markdown.py (hunk 1 above)
Comment (annalee) [inline peacecorps/peacecorps/templatetags/markdown.py]: ah, good call. I'll remove.

**Comment by cmc333333** [comment]
Looks good (but yay automated tests!)
